src/desktop/analytics/main_layout.js

```diff
@@ -3,27 +3,66 @@
 // or any other actions that occur on each page.
 //
 
+import { data as sd } from 'sharify'
+
 // Track pageview
-analytics.page({path: location.pathname}, {integrations: {'Marketo': false}})
+analytics.page(
+  { path: location.pathname },
+  { integrations: { Marketo: false } }
+)
+
+window.addEventListener('load', function() {
+  if (
+    !(window.performance && window.performance.timing) ||
+    !sd.TRACK_PAGELOAD_PATHS
+  ) {
+    return
+  }
+
+  _.each(sd.TRACK_PAGELOAD_PATHS.split('|'), (path) => {
+    if (window.location.pathname.split('/')[1] === path) {
+      window.setTimeout(function() {
```

Comment:
Interesting use of `setTimeout` with a `0` arg. I guess b/c you want to fire off all of these immediately but without blocking other stuff?